package.json

```diff
@@ -43,12 +43,12 @@
     "passport-jwt": "4.0.0",
     "pdf-lib": "1.11.1",
     "pdf2pic": "2.1.2",
-    "pg": "8.4.0",
+    "pg": "^8.4.0",
```

Comment:
same here

Reply:
I am so confused as to why these were added.